README.md

```diff
@@ -76,6 +76,7 @@ Awesome software projects sub-categorized by focus.
 - [Stress2Grid](http://pmd.gfz-potsdam.de/wsm/showshort.php?id=escidoc:2112906) – ![matlab](media/icon/matlab.png) Two concepts to calculate the mean SHmax orientation
 - [QGIS](#platforms) – GIS platform see [Platforms](#platforms)
 - [Verde](https://github.com/fatiando/verde) – ![Python](media/icon/python.png) processing spatial data to regular grids
+- [MB-System](https://github.com/dwcaress/MB-System) - MB-System is an open source software package for the processing and display of bathymetry and backscatter imagery data derived from multibeam, interferometry, and sidescan sonars.
```

Comment:
Please follow the PR template, it would avoid many reviews. The following changes are still necessary:

- [ ] Remove "MB Systems" from description
- [ ] Remove "open source software" from description
- [ ] Fix the dash
- [ ] Add icon for "C" language
- [ ] Remove "."